lib/team_api/canonicalizer.rb

```diff
@@ -9,8 +9,12 @@ class Canonicalizer
     # Canonicalizes the order and names of certain fields within site_data.
     def self.canonicalize_data(site_data)
       sort_collections site_data
```

Comment:
I tend to prefer the parenthesis-less version of method calling where it's allowed.